caffe2/onnx/backend.cc

```diff
@@ -199,6 +200,17 @@ template <> float OnnxAttributes::get(const std::string &key) const {
   return value;
 }
 
+template <> const std::string*
```

Comment:
Any reason why we return a pointer instead of a copy or a const reference?

Reply:
For the performance purpose, I avoid returning copy. 
To be consistent with TensorProto* case at line 239, use const pointer here.